tests/helpers/class-wc-helper-token.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Token helpers.
+ *
+ * @package WooCommerce/Tests
+ */
+
+/**
+ * Class WC_Helper_Token.
+ *
+ * This helper class should ONLY be used for unit tests!.
+ */
+class WC_Helper_Token {
+
+	/**
+	 * Create a token.
+	 *
+	 * @param string $payment_method Token payment method.
+	 * @param int    $user_id        ID of the token's user, defaults to get_current_user_id().
+	 * @param string $gateway        Token's Gateway ID, default to WC_Payment_Gateway_WCPay::GATEWAY_ID
+	 */
+	public static function create_token( $payment_method, $user_id = null, $gateway = WC_Payment_Gateway_WCPay::GATEWAY_ID ) {
+		$token = new WC_Payment_Token_CC();
+		$token->set_token( $payment_method );
+		$token->set_gateway_id( $gateway );
+		$token->set_user_id( $user_id ?? get_current_user_id() );
+		$token->set_card_type( 'visa' );
+		$token->set_last4( '4242' );
+		$token->set_expiry_month( 6 );
+		$token->set_expiry_year( 2026 );
```

Comment:
Since the card is never really used, it shouldn't cause any issues if the token expiry date has passed. However, if WC starts validating the expiry date before creating an already expired token, they could start failing. Just to be safe, I went with the `current year + 1` approach in https://github.com/Automattic/woocommerce-payments/pull/875/commits/f6703f75e8f281f7e0b44db227b6802371b2916e.